src/media/Sound.js

```diff
@@ -1517,6 +1517,9 @@
 			}
 			SoundJS.removeSound(list[i]);
 		}
+
+		//This is to release the audio binary that remains in the jsArrayBufferData.
+		SoundJS.removeAllSounds();
```

Comment:
We probably don't want to unload all sounds here - removeAllSounds() could be called in unloadAll(), but here we want to only unload the list items, and nothing else - we wouldn't want to try to play a sound that we think is loaded, but SoundJS does not.